src/components/experiment-tracking/tooltip/tooltip.scss

```diff
@@ -69,6 +70,22 @@ $triangle-size: 10px;
   }
 }
 
+.tooltip-arrow--right-middle {
```

Comment:
i want to re-use this component as it's built specifically for the ET, however since the design is slightly changed with the arrow is displayed in the middle of the box, I need to introduce this new class name. 

Not sure if there's a better way to do it as I don't like adding more classnames :(

Reply:
Why don't you like adding new class names?

Reply:
hmm i guess it's okay if it's just styling, i just hope that this component won't end up having loads of different classnames, eg: top-right, left-middle etc